src/status_im/contexts/chat/group/details/view.cljs

```diff
@@ -15,6 +14,8 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(def ^:private maximum-selectable-contacts constants/max-group-chat-participants)
```

Comment:
What is the purpose of defining maximum-selectable-contacts?

Reply:
Good catch - I initially added the `(dec)` to exclude the owner account from the selection count, but since we're displaying the owner as well, that adjustment became unnecessary. I missed removing it afterward. Thanks for pointing it out! I've cleaned it up now.